src/runtime/lpython_builtin.py

```diff
@@ -143,6 +143,24 @@ def str(x: i32) -> str:
         result += rev_result[pos]
     return result
 
+@overload
+def str(x :f32) -> str:
+    """
+    Return the string representation of an float 'x'.
+    """
+    res :str
+    res = ''
+    ii: i32
+    ii = int(x)
+    res += str(ii)
+    float_part: f32
+    float_part = x - ii
+    if float_part != 0:
+        res += '.'
+        res += str(int(float_part * 10000))
+
+    return res
```

Comment:
Is this function needed at all? It seems the frontend inserts RealToCharacter Cast, and then the backend implements this by calling `_lfortran_float_to_str`.

Reply:
yes, you're right i leave it becasue i wasn't sure from casting approach.